doc/rst/language/spec/tuples.rst

```diff
@@ -756,43 +725,58 @@ where a comma-separated list of components is valid.
 Value Tuples and Referential Tuples
 -----------------------------------
 
-Throughout the next few sections, the terms referential tuple and value
-tuple are used frequently to describe two different ways that tuples can
-capture elements.
+The terms referential tuple and value tuple describe two different ways
+that tuples capture their elements.
 
-Tuple expressions or tuple arguments with default argument intent are two
-examples of referential tuples. They store elements by reference where it
-makes sense to do so. Referential tuples may be viewed as analogous to a
-group of function arguments that each have default argument intent.
+A *value tuple* stores all its elements by value and may store elements
+copy-initialized from another tuple. A value tuple is analogous to
+a group of formal arguments that each have the ``in`` intent.
+Value tuples include:
 
-Tuple variables or tuple arguments with ``in`` intent are two examples of
-value tuples. They store all elements by value and may store elements copy
-initialized from another tuple. Value tuples may be viewed as analogous to
-a group of function arguments that each have the ``in`` intent.
+- tuple variables
+- formal arguments with ``in`` intent that have a tuple type
+- tuples returned from functions with the default or ``const`` return intent
 
-In short, some or all of the elements of a referential tuple may be
-references, while a value tuple will never contain a reference.
+A *referential tuple* stores its elements by value or by reference,
+as determined by the default argument intent of the element's type:
+
+-  If the default intent is ``const ref`` or ``ref``,
+   the referential tuple's component is a reference.
+
+-  Otherwise, the tuple's component stores a value of the component's type.
```

Comment:
I might say "otherwise, the component is stored by value, as though it had the `in` intent".